beeai/agents/backport_agent.py

```diff
@@ -143,17 +143,22 @@ def create_backport_agent(_: list[Tool], local_tool_options: dict[str, Any]) ->
 
 
 def get_unpacked_sources(local_clone: Path, package: str) -> Path:
-    glob_patterns = (
-        f"{package}-*-build/*-*/",
-        f"{package}-*-build/{package}*/"
-    )
-    for p in glob_patterns:
-        unpacked_sources = list(local_clone.glob(p))
-        if len(unpacked_sources) == 1:
-            break
-    else:
-        raise ValueError(f"Expected exactly one unpacked source, got {unpacked_sources}")
-    return unpacked_sources[0]
+    package_prefix = package.rstrip('0123456789')  # glib2 -> glib
+
+    # in c9s/c10s, the unpacked sources are directly in the local clone, without -build suffix
```

Comment:
dear lord... guess we are not done here and will likely find more edge cases here

Reply:
yes, I want to test this more locally also

Reply:
Well, that actually makes things way easier for us:

```python
from specfile import Specfile
spec = Specfile(local_clone / f"{package}.spec")
return local_clone / spec.expand("%{buildsubdir}")
```

Reply:
updated, PTAL